subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleBuildInstantExecutionSmokeTest.groovy

```diff
@@ -44,14 +44,9 @@ class GradleBuildInstantExecutionSmokeTest extends AbstractSmokeTest {
     }
 
     def "can build gradle with instant execution enabled"() {
-
         given:
         new TestFile("build/gradleBuildCurrent").copyTo(testProjectDir.root)
 
-        and:
```

Comment:
Since we're testing Gradle build here, which requires Java 11, we can set the precondition to JDK11, then use the same JVM as test JVM. WDYT @eskatos ?

Reply:
Building Gradle requires JDK >= 11 and <= 12 because 13 can't target Java 6 anymore so 14 is out.

Reply:
OK, so can we keep the `AvailableJavaHomes.getAvailableJdks` here and move instantSmokeTests back to ReadyForMerge?

Reply:
Yes, leaving this test unchanged and moving the `instantSmokeTest` jobs to `Ready For Merge` would work 👍